Corona-Warn-App/build.gradle

```diff
@@ -329,6 +329,7 @@ dependencies {
     implementation 'com.google.dagger:dagger-android-support:2.28.1'
     kapt 'com.google.dagger:dagger-compiler:2.28.1'
     kapt 'com.google.dagger:dagger-android-processor:2.28.1'
+    kaptTest 'com.google.dagger:dagger-compiler:2.28.1'
```

Comment:
So dagger generates code for classes that only exist in test packages.

Reply:
very helpful hints in this PR - why not put them as comments to the code for future generations to also be able to see it?